### PR TITLE
[Feature] Improve look and quality of FAQ

### PR DESCRIPTION
**Pull request by mfraezz**
Purpose
=======
Closes https://github.com/CenterForOpenScience/osf.io/issues/3350
h/t @jinluyuan for writing up the overview of additions/fixes
screen shot 2015-07-07 at 3 27 39 pm 2

Changes
=======
FAQ Page updated
* Side nav-bar added
* UI Improved
* Questions added / fixed

Side Effects
==========
Fewer helpdesk emails maybe?

### CODE REVIEW

**Comment by jinluyuan** [comment]
Looks pretty! The nav bar would be better on the left side to match the style of other pages' nav bars, like Getting Started.

**Comment by mfraezz** [comment]
Switched it to the left:
screen shot 2015-07-07 at 3 54 42 pm 2

**Comment by jinluyuan** [comment]
Also closes https://github.com/CenterForOpenScience/osf.io/issues/3415

File: website/templates/public/pages/faq.mako [hunk 1]
```diff
@@ -1,122 +1,157 @@
...
+        
+        <div class="col-xs-12 col-sm-9">
+            <div class="page-header">
+                <h1>Frequently Asked Questions</h1>
+            </div>
+
+            <div id="about" class="anchor">
+                <h3>About</h3>
+                <div class="row m-l-lg">
+                    <h4 style="margin-bottom: -5px;">How much does the OSF service cost?</h4>
```
Comment (sloria) [inline website/templates/public/pages/faq.mako]: Use a CSS rule rather than inline styling to reduce duplication.

**Comment by sloria** [comment]
Once the above CSS change is made and a language review is done, this is good to merge.

**Comment by mfraezz** [comment]
CSS change made, awaiting language review @lbanner

**Comment by lbanner** [comment]
Just a few changes @mfraezz!

Paragraph at line 35: “the OSF” is presently proper usage - need to add “The” to sentences 2, 3, and 4.

Line 58: Replace “to” with “that you" : “This FAQ page will only cover a few of the most basic questions about the OSF, so it’s highly recommended ^that you^ read through Getting Started if you have any issues.”

Line 73: Add space: “Registering is^space^an optional feature of the OSF.”

Line 79 3rd sentence: Add “the” : “Then click ^the^ ”Create DOI / ARK" link, located in the block of text below the project title.”

Line 100: Add “the” : “When you click "Forgot your password," ^the^ OSF sends you a new random”

Line 103: Add “the” : “Data and materials posted on ^the^ OSF are not yet encrypted, unless you encrypt”

**Comment by GaryKriebel** [comment]
Were the style changes checked in.... not seeing them:
screen shot 2015-07-08 at 10 24 40 am

Also, in other places on the site where we have a left hand nav, it is surrounded by a thin box as shown below.
screen shot 2015-07-08 at 10 25 46 am

**Comment by mfraezz** [comment]
`inv assets` had crashed without me noticing so some style overrides weren't displayed for me. https://github.com/CenterForOpenScience/osf.io/pull/3486 fixes this.

**Pull request merged**